### DISCUSSION TITLE
Baby Buddy running within Home assistant

### DISCUSSION THREAD

**Discussion by rizzol19** [babybuddy/babybuddy#594] [Q&A]
Hi,

I have installed baby buddy within the home assistant that is running on a virtual box. I can not figure out what the IP address of baby buddy is within home assistant. I added HACS and the baby buddy repo. However, when I go to install the baby buddy integration, I can not get it to work as I receive errors for unable to connect or invalid authentication(this is when I use the home assistant IP, guessing it is trying to access home assistant via API). I have also looked at all the API settings in baby buddy with IP addresses and none of those work either. Any help on this would be great.

Thanks!

**Comment by cdubz** [comment]
Recommend you take a look at https://github.com/jcgoette/baby_buddy_homeassistant and https://github.com/OttPeterR/addon-babybuddy. There is likely better support for Home Assistant issues there.
